addon/index.js

```diff
@@ -56,6 +128,11 @@ export class EmberChangeset extends BufferedChangeset {
     return safeSet(obj, key, value);
   }
 
+  @dependentKeyCompat
+  get mirror() {
+    return this[VIRTUAL_MIRROR];
+  }
```

Comment:
Is this meant to be access anywhere?  I don't see `this.mirror` anywhere.

Reply:
Mainly for public access, for the classic computeds case